appcues/src/main/java/com/appcues/ui/primitive/OptionSelectPrimitive.kt

```diff
@@ -151,22 +151,22 @@ private fun List<OptionSelectPrimitive.OptionItem>.ComposeSelections(
         when (controlPosition) {
             LEADING -> Row(verticalAlignment = Alignment.CenterVertically) {
                 selectMode.Compose(isSelected, selectedColor, unselectedColor, accentColor) { updateSelection(it.value, !isSelected) }
-                it.content.Compose()
+                it.contentView(isSelected).Compose()
             }
             TRAILING -> Row(verticalAlignment = Alignment.CenterVertically) {
-                it.content.Compose()
+                it.contentView(isSelected).Compose()
                 selectMode.Compose(isSelected, selectedColor, unselectedColor, accentColor) { updateSelection(it.value, !isSelected) }
             }
             TOP -> Column(horizontalAlignment = Alignment.CenterHorizontally) {
                 selectMode.Compose(isSelected, selectedColor, unselectedColor, accentColor) { updateSelection(it.value, !isSelected) }
-                it.content.Compose()
+                it.contentView(isSelected).Compose()
             }
             BOTTOM -> Column(horizontalAlignment = Alignment.CenterHorizontally) {
-                it.content.Compose()
+                it.contentView(isSelected).Compose()
                 selectMode.Compose(isSelected, selectedColor, unselectedColor, accentColor) { updateSelection(it.value, !isSelected) }
             }
             HIDDEN -> Box(modifier = Modifier.clickable { updateSelection(it.value, !isSelected) }) {
-                it.content.Compose()
+                it.contentView(isSelected).Compose()
```

Comment:
these are the normal cases when rendering selection content in a vertical or horizontal list